app/src/main/java/com/fstyle/structure_android/screen/userdetail/UserDetailPresenter.java

```diff
@@ -1,5 +1,11 @@
 package com.fstyle.structure_android.screen.userdetail;
 
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.repository.UserRepository;
+import com.fstyle.structure_android.utils.rx.BaseSchedulerProvider;
+import com.google.common.base.Preconditions;
+import io.reactivex.functions.Consumer;
```

Comment:
ERROR: Unused import: io.reactivex.functions.Consumer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports